doc/specs/Configuration object for profiles.md

```diff
@@ -0,0 +1,125 @@
+---

```

Comment:
nit: rename file to include original issue ID -- also consider including the word appearance somewhere in the file